crates/biome_lsp/src/handlers/analysis.rs

```diff
@@ -242,6 +242,7 @@ fn fix_all(
     let fixed = session.workspace.fix_file(FixFileParams {
         path: biome_path,
         fix_file_mode: FixFileMode::SafeFixes,
+        suppress: false,
```

Comment:
TODO: Weird...

Reply:
This isn't weird, but it's a normal side-effect of our architecture. Biome, internally, relies on `Workspace` trait that is used both by the CLI and LSP, which are considered as "clients" of our architecture.